Doc changes of ROS Noetic installation and Quick Start guide

laukikk opened the pull request:
The following changes would be reflected:

## ROS Noetic Installation Docs:
- [x] Spelling check
- [x] changes in the manual installation of ROS for [ROS Wiki](http://wiki.ros.org/noetic/Installation/Ubuntu)
- [x] VNC for chrome is deprecated mentioned
- [x] behavior_studio is now behavior_metrics

## Quick Start guide
- [x] Commented the link for the tutorial for "Train a Brain". (Yet to develop)
- [x] bash commands update

this commit fixes #333

laukikk committed:
Doc changes of ROS Noetic installation and Quick Start guide

laukikk commented:
I would be making another PR to add the documentation of "Make a Brain", linking it to #48.

sergiopaniego commented on _pages/install/ros_noetic.md:
```diff
@@ -247,10 +248,11 @@ docker run --gpus all -dit --name behavior-metrics-noetic \
 ### Using VNC to visualize container <a name="vnc"></a>
 
-To connect to our container [VNC viewer for chrome](https://chrome.google.com/webstore/detail/vnc%C2%AE-viewer-for-google-ch/iabmpiboiopbgfabjmgeedhcmjenhbla?hl=en) (recommended) or [RealVNC](https://www.realvnc.com/en/) can be installed to access the GUI through the port 5900.
+To connect to our container [VNC viewer for chrome](https://chrome.google.com/webstore/detail/vnc%C2%AE-viewer-for-google-ch/iabmpiboiopbgfabjmgeedhcmjenhbla?hl=en) (depricated) or [RealVNC](https://www.realvnc.com/en/) can be installed to access the GUI through the port 5900.
```
Since the link is broken, please remove it :smile:

sergiopaniego commented:
> I would be making another PR to add the documentation of "Make a Brain", linking it to #48.

Nice idea! We could first finish this one before continuing

laukikk commented:
Yes, let me know if there's anything else. You can see the deployment of my PR [here](https://laukikk.github.io/BehaviorMetrics/).

laukikk committed:
Added doc for Training a Brain

laukikk commented:
I've added the documentation for `Training a Brain`. A part of the documentation was already available but not correctly linked. Its now fixed and the documentation is also updated.

this commit fixes #48

sergiopaniego commented:
Thanks for the updated contribution, it looks great!

I can still see the broken link that I pointed on the previous review. Could you review it, removing that link? Then, we can accept the PR :smile:

laukikk commented:
Hey @sergiopaniego, my [first commit](https://github.com/JdeRobot/BehaviorMetrics/pull/334/commits/6f8c0fb0bc5d7bfaac83e043e3a83fc97f1405f9) on this PR, removed the link as there were no docs present, but in the [next commit](https://github.com/JdeRobot/BehaviorMetrics/pull/334/commits/c2ac63c6cbc6f892ca8900ea04fcc11c36ac1b4a), I added the docs for `Training a Brain` and that is why I brought that link bank, it is [not broken](https://laukikk.github.io/BehaviorMetrics/quick_start/tutorial) anymore. If you want I could just make a new PR altogether for fixing it, instead of appending the commit in this PR.

sergiopaniego commented:
Hi @laukikk,
I probably explained myself wrong. 
I was talking about the link that points to https://chrome.google.com/webstore/detail/vnc%C2%AE-viewer-for-google-ch/iabmpiboiopbgfabjmgeedhcmjenhbla?hl=en since this link returns a 404 it should be removed.

laukikk committed:
removed broken link of VNC for chrome

sergiopaniego commented:
Thanks for the update!
We can merge the branch now
